samples/client/petstore/typescript-angular/api.module.ts

```diff
@@ -0,0 +1,9 @@
+import * as api from './api/api';
```

Comment:
Is there a specific reason why you do the ``` * as api``` import instead of ```import {PetApi, StoreApi,...} from './api/api';```

We also made this choice at first in angular2 API's but due to problems we needed to change this later on.